src/app/public/styles/_forms.scss

```diff
@@ -27,7 +27,7 @@
 }
 
 input.ng-invalid.ng-touched,
-select.ng-invalid.submitted {
+select.ng-invalid.ng-touched {
```

Comment:
Should we keep the old `.submitted` state as well? Would this change anything for consumers if we remove it?

Reply:
In looking through examples it does not appear that `submitted` is actually an Angular class. So I'm not sure where that was ever coming from? I think we are ok to remove it.

Reply:
Sounds good.